deploy.sh

```diff
@@ -159,8 +152,9 @@ done
 
 echo -e "\n\n${bold}Cloning project repo...${normal}\n\n"
 
-rm -rf "$REPONAME"
-git clone "https://github.com/sisoputnfrba/${REPONAME}.git"
+if [ ! -d "${REPONAME}" ]; then
+    git clone "https://github.com/sisoputnfrba/${REPONAME}.git"
```

Comment:
If we add `--recurse-submodules --shallow-submodules --depth 1` here, do we see any performance improvement as well?

Also, deploying `main` branch should be a requirement imo, so I would even include `--branch main`.